packages/validator/test/unit/utils/params.test.ts

```diff
@@ -12,20 +12,23 @@ const testCases: {name: string; items: [IChainConfig, Record<string, string>]}[]
   {name: "nimbus", items: [networksChainConfig.ropsten, nimbusRopstenConfig]},
 ];
 
+/* eslint-disable @typescript-eslint/naming-convention */
+
 describe("utils / params / assertEqualParams", () => {
   it("default == default", () => {
     const chainConfigJson = chainConfigToJson(chainConfig);
     assertEqualParams(chainConfig, chainConfigJson);
   });
 
   it("default != other", () => {
-    const chainConfigJson = chainConfigToJson(chainConfig);
+    const ALTAIR_FORK_EPOCH = 10;
```

Comment:
Make test deterministic test independent of current values of the default chain spec